### DISCUSSION TITLE
Mac 编译失败的问题

### DISCUSSION THREAD

**Discussion by urourouroooo** [sjtug/SJTUThesis#811] [提问]
<img width="1170" alt="image" src="https://user-images.githubusercontent.com/106020218/169697134-9781b36c-bad0-4273-99d2-01c5a9e705ce.png">
想要解析本科论文的main.tex
已经按照README设置了latexmk-sjtuthesis 但是仍然解析失败

**Comment by skyzh** [comment]
请看一下最新版的 readme。另外出错请提供报错信息，你的截图里什么也看不出来 🤪
